tools/tf2openapi/cmd/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"flag"
+	"github.com/golang/protobuf/proto"
+	"github.com/kfserving/tools/tf2openapi/generator"
+	pb "github.com/tensorflow/tensorflow/tensorflow/go/core/protobuf"
+	"io/ioutil"
+	"log"
+)
+
+var model = flag.String("model", "", "Absolute path of SavedModel file")
+
+func main() {
+	flag.Parse()
+
+	in, err := ioutil.ReadFile(*model)
+	if err != nil {
+		log.Print(*model)
+		log.Fatalln("Error reading file:", err)
+	}
+
+	/** Convert Go struct to inner model */
+	model := UnmarshalSavedModelPb(in)
+
+	/** Schema generation example **/
+	log.Println(generator.GenerateOpenAPI(model))
+}
+
+/**
+Raises errors when model is missing fields that would pose an issue for Schema generation
+ */
+func UnmarshalSavedModelPb(in []byte) pb.SavedModel {
```

Comment:
I returned an object because it would be dereferenced when constructing the internal models (I use objects as inputs to the "constructors" because I didn't want the inputs to the "constructors" to change). However, seeing that the SavedModel could be quite large, maybe I could pass pointers to the TFSavedModel and TFMetaGraph constructors, but return objects for my internal models?